modules/reference/pages/rpk/rpk-security/rpk-security-secret-update.adoc

```diff
@@ -0,0 +1,59 @@
+= rpk security secret update
+// tag::single-source[]
+
+Update an existing secret for your Redpanda Cloud cluster.
+
+Scopes define the areas where the secret can be used. Available scopes are:
+
+- `redpanda_connect`
+- `redpanda_cluster`
+
+You can set one or both scopes on a secret. Updating a secret's scopes will overwrite its current scopes. 
+
+== Usage
+
+[,bash]
+----
+rpk security secret update [flags]
+----
+
+== Examples
+
+To update the value of the secret:
+
+[,bash]
+----
+rpk security secret update --name NETT --value new_value
+----
+
+To update the scope of a secret to both `redpanda_connect` and `redpanda_cluster`:
+
+[,bash]
+----
+rpk security secret update --name NETT2 --value value --scopes redpanda_connect,redpanda_cluster
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for update.
+
+|--name |string |Name of the secret. The name must be in uppercase and can only contain letters, digits, and underscores.
```

Comment:
@andresaristizabal can you change the name of a secret?

Reply:
No, this name is the ID of the secret